### ISSUE TITLE
list index out of range. ST3

### ISSUE THREAD

**Issue by feyyazesat** [SublimeText/CTags#282]
I commanded Ctag Rebuild Tags after a minute the error pop up coming appear and the console present below errors.

```
error: An unknown error occured.
Check the console for info.
Exception in thread Thread-18680:
Traceback (most recent call last):
  File "./threading.py", line 901, in _bootstrap_inner
  File "./threading.py", line 858, in run
  File "ctagsplugin in /home/feyyazesat/.config/sublime-text-3/Installed Packages/CTags.sublime-package", line 119, in run
  File "ctagsplugin in /home/feyyazesat/.config/sublime-text-3/Installed Packages/CTags.sublime-package", line 907, in build_ctags
  File "ctagsplugin in /home/feyyazesat/.config/sublime-text-3/Installed Packages/CTags.sublime-package", line 892, in build_ctags
  File "ctags in /home/feyyazesat/.config/sublime-text-3/Installed Packages/CTags.sublime-package", line 331, in build_ctags
  File "ctags in /home/feyyazesat/.config/sublime-text-3/Installed Packages/CTags.sublime-package", line 365, in resort_ctags
IndexError: list index out of range
```

**Comment by pitosalas** [comment]
I am having this problem as well. Using ST3 on OS X, against Ruby sources. Trying a simple case. 

Traceback (most recent call last):
  File "/Applications/Sublime Text.app/Contents/MacOS/sublime_plugin.py", line 556, in run_
    return self.run(edit)
  File "ctagsplugin in /Users/pitosalas/Library/Application Support/Sublime Text 3/Installed Packages/CTags.sublime-package", line 609, in command
  File "ctagsplugin in /Users/pitosalas/Library/Application Support/Sublime Text 3/Installed Packages/CTags.sublime-package", line 801, in run
  File "ctagsplugin in /Users/pitosalas/Library/Application Support/Sublime Text 3/Installed Packages/CTags.sublime-package", line 794, in get_tags
  File "ctags in /Users/pitosalas/Library/Application Support/Sublime Text 3/Installed Packages/CTags.sublime-package", line 531, in get_tags_dict
  File "ctags in /Users/pitosalas/Library/Application Support/Sublime Text 3/Installed Packages/CTags.sublime-package", line 92, in parse_tag_lines
  File "ctags in /Users/pitosalas/Library/Application Support/Sublime Text 3/Installed Packages/CTags.sublime-package", line 489, in search
  File "ctags in /Users/pitosalas/Library/Application Support/Sublime Text 3/Installed Packages/CTags.sublime-package", line 400, in __lt__
IndexError: list index out of range

**Comment by a06062125** [comment]
I've met the  same question while execute "Rebuild Ctags" command. Is there any solution？

`执行“Rebuild Ctags”时抛以下异常：
···
reloading Packages/User/CTags.sublime-settings
Building CTags for /Users/weimengxi/devspace/cgm_entry/src/web_inf/front/static/js: Please be patient
error: An unknown error occured.
Check the console for info.
Exception in thread Thread-4:
Traceback (most recent call last):
  File "./threading.py", line 901, in _bootstrap_inner
  File "./threading.py", line 858, in run
  File "ctagsplugin in /Users/weimengxi/Library/Application Support/Sublime Text 3/Installed Packages/CTags.sublime-package", line 119, in run
  File "ctagsplugin in /Users/***/Library/Application Support/Sublime Text 3/Installed Packages/CTags.sublime-package", line 907, in build_ctags
  File "ctagsplugin in /Users/***/Library/Application Support/Sublime Text 3/Installed Packages/CTags.sublime-package", line 892, in build_ctags
  File "ctags in /Users/***/Library/Application Support/Sublime Text 3/Installed Packages/CTags.sublime-package", line 331, in build_ctags
  File "ctags in /Users/***/Library/Application Support/Sublime Text 3/Installed Packages/CTags.sublime-package", line 365, in resort_ctags
IndexError: list index out of range
···

**Comment by washaweb** [comment]
same for me

**Comment by WIZARD-CXY** [comment]
me 2

**Comment by pwoytiuk** [comment]
I've gotten the same error in Sublime Test 2.0.2:

Exception in thread Thread-4:
Traceback (most recent call last):
  File ".\threading.py", line 532, in __bootstrap_inner
  File ".\threading.py", line 484, in run
  File ".\ctagsplugin.py", line 119, in run
  File ".\ctagsplugin.py", line 907, in build_ctags
IndexError: list index out of range

**Comment by Mparaiso** [comment]
Same problem here : 

loading symbols from file
Traceback (most recent call last):
  File ".\sublime_plugin.py", line 362, in run_
    return self.run(edit)
  File ".\ctagsplugin.py", line 609, in command
  File ".\ctagsplugin.py", line 801, in run
  File ".\ctagsplugin.py", line 794, in get_tags
  File ".\ctags.py", line 531, in get_tags_dict
  File ".\ctags.py", line 92, in parse_tag_lines
  File ".\ctags.py", line 489, in search
  File ".\ctags.py", line 400, in __lt__
IndexError: list index out of range

**Comment by RogerTsang** [comment]
Same problem.
Try run "ctags -R" on your project directory to get around from it.

**Comment by deathaxe** [comment]
Fixed by 0.5.1
